app/views/users/edit.html.haml

```diff
@@ -11,11 +11,11 @@
   = radio_button_tag :gender, 'Female', class: "form-check input"
   = label_tag :gender, 'Female'
 
-  = label_tag(:weight, 'Weight')
+  = label_tag(:weight, 'Weight (lbs)')
   = number_field_tag(:weight, 'weight', min: 1, required: true)
 
-  = label_tag(:height, 'Height')
-  = number_field_tag(:height, 'height', min: 1, required: true)
+  = label_tag(:height, 'Height (inches)')
+  = select_tag(:height, options_for_select(["53 (4'5)", "54 (4'6)", "55 (4'7)", "56 (4'8)", "57 (4'9)", "58 (4'10)", "59 (4'11)", "60 (5'0)","61 (5'1)", "62 (5'2)", "63 (5'3)", "64 (5'4)", "65 (5'5)", "66 (5'6)", "67 (5'7)", "68 (5'8)", "69 (5'9)", "70 (5'10)", "71 (5'11)", "72 (6'0)", "73 (6'1)", "74 (6'2)", "75 (6'3)", "76 (6'4)", "77 (6'5)", "78 (6'6)", "79 (6'7)", "80 (6'8)", "81 (6'9)", "82 (6'10)", "83 (6'11)", "84 (7'0)", "85 (7'1)", "86 (7'2)", "87 (7'3)", "88 (7'4)", "89 (7'5)"]))
```

Comment:
this adds the changes we originally wanted from the "Add imperial units" back into master